tests/tests.zig

```diff
@@ -1,6 +1,5 @@
 comptime {
     _ = @import("helper.zig");
-    _ = @import("sessions.zig");
```

Comment:
was this intentional?

Reply:
yes, my first commit removed that file because it includes mostly just test for completion which all fail when you have so many snippets. I will have to implement completion tests at some point...